### DISCUSSION TITLE
GH Projects; can't query by author

### DISCUSSION THREAD

**Discussion by chrislujan** [community/community#41753] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Is there a way to filter project issues by `author`? I'd like to be able to add that field as a column and filter project views by the `author` of an issue.

**Comment by rileybroughten** [comment]
Hello @chrislujan! 👋 Currently there is not an `author` field available in Projects, rather only the `assignee` field for the item. It is on our backlog to provide this type of item metadata as fields in Projects, so I will take this back to the team and keep you updated! 👍

**Reply by Shinigami92** [reply]
I want to add me to the interested and waiting list for this feature 👍

**Reply by yanneman** [reply]
same here, this really feels like one of the absolute basic fields we need to filter on

**Reply by agilesteel** [reply]
It's been a year since last response. Any updates?

**Reply by yanneman** [reply]
just makes the board unusable. We're thinking about different platforms just because of little things like these that literally take 10 mins to develop but that are not available

**Comment by vhartikainen** [comment]
Basic fields for search, when are you planning to add this? Really odd that you can search the issue list per author, but not filter the board
